### PR TITLE
Add FreshPorts report

### PR DESCRIPTION
**Pull request by dlangille**

### CODE REVIEW

File: website/content/en/status/report-2024-01-2024-03/freshports.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
+=== FreshPorts: Notification of new packages
+
+Links: +
+link:https://freshports..org[FreshPorts] URL: link:freshports.org[] +
```
Comment (pauamma) [inline website/content/en/status/report-2024-01-2024-03/freshports.adoc:4]: Extra dot before "org".

File: website/content/en/status/report-2024-01-2024-03/freshports.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+FreshPorts tracks the commits and extracts data from the port Makefiles to create a database of information useful to both port maintainers and port users.
+
+For example, link:https://www.freshports.org/security/acme.sh/#packages[] shows the history of the package:security/acme.sh[] port, back to its creation in May 2017.
+Also available are dependencies, flavors, configuration options, and available packages.
+All of this is useful for both users and developers of ports.
+
+==== Notification: New Package Available
+
+One of the original features of FreshPorts is notification of ports updates. You
+can create a list of ports and receive notifcations about those ports. This
```
Comment (pauamma) [inline website/content/en/status/report-2024-01-2024-03/freshports.adoc:21]: s/notifcations/notifications/

File: website/content/en/status/report-2024-01-2024-03/freshports.adoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+new feature can also notify when a new pacakge is available for that port.
```
Comment (pauamma) [inline website/content/en/status/report-2024-01-2024-03/freshports.adoc:22]: s/pacakge/package/

**Review by pauamma — APPROVED** [review]
Approved once these typos are fixed. (Fixable on commit.)

**Comment by dlangille** [comment]
Good changes, thanks.

File: website/content/en/status/report-2024-01-2024-03/freshports.adoc (hunk 1 above)
Comment (grahamperrin) [inline website/content/en/status/report-2024-01-2024-03/freshports.adoc:4]:
```suggestion
link:https://freshports.org/[FreshPorts] URL: link:https://freshports.org/[] +
```

- URL: link:https: (the weird norm)
- plus the correction from @pauamma

File: website/content/en/status/report-2024-01-2024-03/freshports.adoc (hunk 3 above)
Comment (grahamperrin) [inline website/content/en/status/report-2024-01-2024-03/freshports.adoc:22]:
```suggestion
One of the original features of FreshPorts is notification of ports updates.
You can create a list of ports and receive notifications about those ports.
This new feature can also notify when a new package is available for that port.
```

- one sentence per line
- plus the corrections from @pauamma

File: website/content/en/status/report-2024-01-2024-03/freshports.adoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+The database does not change very often and requires little maintenance compared to the applications and OS.
+The website pretty much runs itself.
+From time to time, a change to the FreeBSD ports infrastructure breaks something or requires a modification, but there is rarely any urgency to fix that.
+This is not a huge time commitment.
+There is a lot of learning.
+While not a complex application, FreshPorts is also not trivial.
+
+To contribute, please join the
+link:https://lists.freshports.org/mailman/listinfo/freshports-coders[]
+mailing list and let us know what you'd like to help with.
```
Comment (grahamperrin) [inline website/content/en/status/report-2024-01-2024-03/freshports.adoc:52]:
```suggestion
To contribute, please join the link:https://lists.freshports.org/mailman/listinfo/freshports-coders[] mailing list and let us know what you'd like to help with.
```

File: website/content/en/status/report-2024-01-2024-03/freshports.adoc [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+Also available are dependencies, flavors, configuration options, and available packages.
+All of this is useful for both users and developers of ports.
+
+==== Notification: New Package Available
+
+One of the original features of FreshPorts is notification of ports updates. You
+can create a list of ports and receive notifcations about those ports. This
+new feature can also notify when a new pacakge is available for that port.
+The use case: a known security vuln has been patched. FreshPorts will tell
+you the port has been patched, and then you wait for the package.
```
Comment (grahamperrin) [inline website/content/en/status/report-2024-01-2024-03/freshports.adoc:24]:
```suggestion
The use case: a known security vuln has been patched.
FreshPorts will tell you the port has been patched, and then you wait for the package.
```

**Review by grahamperrin — CHANGES_REQUESTED** [review]

**Comment by dlangille** [comment]
Have I done all the things to bring in all the things?

**Comment by grahamperrin** [comment]
> Have I done all the things to bring in all the things?

I guess so, thanks, however it's not for me to approve. 

----

Note to self: the _suggested changes_ feature brings simplicity, however in a case such as this – where I use the feature to suggest changes that originated from someone else – the commits do not show the other person's name. Sorry @pauamma

**Review by lsalvadore — APPROVED** [review]
Just two small more changes and this is ready for commit. Feel free to commit it yourself or I can commit it for you if you prefer.

Thanks for your report.

**Comment by dlangille** [comment]
I hope I've covered all pending changes.

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**